apps/site/assets/css/_stop-page.scss

```diff
@@ -320,3 +320,17 @@
   background-color: aqua;
   height: 160px;
 }
+
+.route-schedules-heading,
+.bike-heading {
+  font-family: 'Helvetica Neue';
+  font-size: 1.3rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.6rem;
+}
+
+.bike-heading {
+  font-size: 1rem;
+  line-height: 1.25;
+}
```

Comment:
Don't the heading tags already have the right styling as-is? Most of these properties shouldn't need customization.

Reply:
@thecristen For clarity should I just remove all this or just keep the font-size? The font-size is the only difference between designs and default styling as far as I can tell.

Reply:
Using heading styles in the mockups generally implies using the existing heading styles as-is, but I've seen the mocked designs be inconsistent in which H2/3 it uses across different screens... so let's remove it all for now, and check in on this when we do our eventual design review.